public/app/features/dashboard/dashgrid/DashboardLibrary/CommunityDashboardSection.tsx

```diff
@@ -32,7 +32,7 @@ interface Props {
 }
 
 // Constants for community dashboard pagination and API params
-const COMMUNITY_PAGE_SIZE = 9;
+const COMMUNITY_PAGE_SIZE = 51;
```

Comment:
The PR description mentions "max 50 dashboards" but this is set to 51. Is this intentional (to account for filtering)? or should this be `50`?